docs/OnTrack/Voice Verification/Architecture-Document.md

```diff
@@ -0,0 +1,52 @@
+# Architecture Document | Voice Verification for OnTrack Delivery
+## Introduction
+### Purpose
+This document provides a comprehensive architectural overview of the system, using a few different architectural views to depict different aspects of the system. It is intended to capture and convey the significant architectural decisions which have been made on the system. 
+### Scope
+This Architecture Document provides an architectural overview of Voice Verification System. The Voice Verification System is being developed to address the issues concerning contract cheating within online learning management platforms. 
+
+## Architectural Goals and Constraints
+- Front-end components should comply with existing OnTrack requirements. 
+- Students can register their voice on OnTrack using the Speaker Verification system. 
+- Speaker Verification system must have compatibility for voice recording across multiple browsers. 
+- Upon task submission, the attached voice file is analysed for verification. 
+- Deployed on the OnTrack instance in a docker container format. 
+- Support for gathering both front and backend telemetry should be present in the system to allow for analysis of user interaction, and system performance. 
+- System should adhere to existing OnTrack privacy/compliance requirements in addition to existing OnTrack security requirements. 
+
+## Use-Case View
+### Architecturally Significant Use Cases
+![Figure 1: Architecturally Significant Use Cases](https://github.com/thoth-tech/documentation/blob/89e17ecb3033c6491c9e7d7ee161198b2ee04653/docs/OnTrack/Voice%20Verification/Images/Use%20Cases.png)
+
+
+- Enrol the voiceprint: This use case allows a student to register a voiceprint for later verification 
```

Comment:
Would be good to format these each as a user story and then explain them with the points above